README.md

```diff
@@ -1,181 +1,134 @@
-# Chrome Apps samples
+# Chrome Extensions samples
 
-Official samples for Chrome Apps platform. If you want to learn about the platform, you can:
+Official samples for Chrome Extensions (and the [deprecated](https://developer.chrome.com/apps/migration) Chrome Apps platform).
```

Comment:
I think instead of the parenthetical it might be good to say "Official samples for Chrome Extensions and the Chrome Apps platform" and below that add a note that Chrome apps are deprecated and link to the blog post to 'learn more'.

Reply:
Great.